demisto_sdk/commands/validate/validators/RM_validators/RM108_is_image_path_valid.py

```diff
@@ -115,7 +122,7 @@ def verify_relative_saved_in_doc_files(self, content_item) -> str:
         invalid_links = [
             rel_img
             for rel_img in relative_images
-            if not re.match(DOC_FILE_IMAGE_REGEX, rel_img)
+            if not re.search(DOC_FILE_IMAGE_REGEX, rel_img)
```

Comment:
Why are we changing this?

Reply:
The change from re.match to re.search was necessary because re.match only checks for the pattern at the start of the string, whereas re.search looks for the pattern anywhere in the string. (for example ../../doc_files / ../../../doc_files /doc)

This ensures all occurrences of the pattern are correctly matched, regardless of where they appear in the string.